Sources/SPTPersistentCacheTypeUtilities.h

```diff
@@ -29,3 +30,13 @@
  * @return The value as an `uint64_t`.
  */
 uint64_t spt_uint64rint(double value);
+
+
+/**
+ *  Executes the debug callback safely avoiding a crash if it is set to nil.
+ *
+ *  @param debugMessage The debug message.
+ *  @param debugCallback The callback block to execute safely.
+ */
+void SPTPersistentCacheSafeDebugCallback(NSString *debugMessage,
```

Comment:
This is a bit of a nitpick but `SPTPersistentCacheSafeDebugCallback()` is not a _type utility_. Which this header is for according to its name. How about moving this function to its own file? E.g. `SPTPersistentCacheDebugging.h` or something like that?